sale_order_line_menu/models/sale_order_line.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Moduon Team
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    date_order = fields.Datetime(
+        related="order_id.date_order",
+        readonly=True,
+        store=True,
+        index=True,
+    )
+    account_analytic_id = fields.Many2one(
+        related="order_id.analytic_account_id",
+        readonly=True,
+        store=True,
+        index=True,
+    )
```

Comment:
I think we should fix the typo and rename field the same as in sale.order:
 account_analytic_id -> analytic_account_id

of cause this needs also to be updated in the views.